metadata-service/factories/src/main/java/com/linkedin/gms/factory/kafka/KafkaEventConsumerFactory.java

```diff
@@ -35,6 +35,24 @@ public class KafkaEventConsumerFactory {
   @Value("${kafka.bootstrapServers}")
   private String kafkaBootstrapServers;
 
+  @Value("${kafka.protocol}")
```

Comment:
I don't believe this is required. The spring kafka library is used and is using Spring's kafka implementation which exposes the configuration of these properties via Spring's environment variables.

For example, the consumer truststore [location](https://docs.spring.io/spring-boot/docs/2.5.14/reference/html/application-properties.html#application-properties.integration.spring.kafka.consumer.ssl.key-store-location)

SPRING_KAFKA_CONSUMER_SSL_KEY-STORE-LOCATION=
or if common with the producer
SPRING_KAFKA_SSL_TRUST-STORE-LOCATION=

Is there a need to add the code and configuration? Maybe just documentation around using the Spring environment variables?